crates/fluvio-cli/src/connector/mod.rs

```diff
@@ -92,40 +94,146 @@ pub struct ConnectorConfig {
     name: String,
     #[serde(rename = "type")]
     type_: String,
+
     pub(crate) topic: String,
     pub(crate) version: Option<String>,
+
     #[serde(default)]
     parameters: BTreeMap<String, String>,
+
     #[serde(default)]
     secrets: BTreeMap<String, SecretString>,
+
+    #[serde(default)]
+    producer: Option<ProducerParameters>,
+
+    #[serde(default)]
+    consumer: Option<ConsumerParameters>,
+}
+
+#[derive(Debug, Clone, Deserialize)]
+pub struct ConsumerParameters {
+    #[serde(default)]
+    partition: Option<i32>,
```

Comment:
I'm tempted to add a `offset` field here but it's unclear how it should be specified. Thoughts?

Reply:
should be the same as CLI but can be add later